eng/ci/templates/jobs/run-unit-tests-linux.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+- job: RunUnitTests_Linux
```

Comment:
You can remove the separate files for windows and linux and include pool as a parameter:

``` yml
parameters:
  - name: os
     type: string
     values:
       - windows
       - linux
.
.
.
  - ${{ if eq(parameters.os, 'windows') }}:
    pool:
      name: 1es-pool-azfunc
      image: 1es-windows-2022
      os: windows
  - ${{ elseif eq(parameters.os, 'linux') }}:
     pool:
       name: 1es-pool-azfunc
       image: 1es-ubuntu-22.04
       os: linux
```

Can also us `${{ parameters.os }}` expression as part of the display name and job name

Reply:
Or maybe strategy/matric will work here also:
https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/jobs-job-strategy?view=azure-pipelines

Reply:
Oooo I like this idea, will do